guides/getting-started/set-up/index.md

```diff
@@ -125,9 +125,11 @@ required.
 Sets up an HTTP outbound and inbound transport.
 
 ```typescript showLineNumbers
-import { HttpOutboundTransport, WsOutboundTransport, HttpInboundTransport } from '@aries-framework/core'
+import { WsOutboundTransport, HttpOutboundTransport } from "@aries-framework/core";
```

Comment:
We should probably also update this to use the snippets (it's there to avoid incorrect usages like this)

@blu3beri I see you added the snippets in https://github.com/hyperledger/aries-javascript-docs/pull/45, but didn't use them in the markdown?

Reply:
ping @blu3beri

Reply:
Ah I will remind myself to pick this up tomorrow. Dumb mistake.